app/frontend/src/features/messages/surfing/HostRequestView.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, BoxProps, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import * as React from "react";
+import { useCallback, useEffect, useState } from "react";
+import { useStore } from "react-redux";
+import Alert from "../../../components/Alert";
+import CircularProgress from "../../../components/CircularProgress";
+import { Message } from "../../../pb/conversations_pb";
+import { HostRequest } from "../../../pb/requests_pb";
+import { service } from "../../../service";
+import { fetchUsers, getUser } from "../../userCache";
+import MessageList from "../messagelist/MessageList";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface HostRequestViewProps extends BoxProps {
+  hostRequest: HostRequest.AsObject;
+}
+
+export default function HostRequestView({ hostRequest }: HostRequestViewProps) {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const [messages, setMessages] = useState<Message.AsObject[]>([]);
+  const fetchMessages = useCallback(async () => {
+    setLoading(true);
+    try {
+      const messages = await service.requests.getHostRequestMessages(
+        hostRequest.hostRequestId
+      );
+      setMessages(messages);
+    } catch (error) {
+      setError(error.message);
+    }
+    setLoading(false);
+  }, [hostRequest.hostRequestId]);
+
+  useEffect(() => {
+    dispatch(fetchUsers({ userIds: [hostRequest.fromUserId, hostRequest.toUserId] }));
```

Comment:
It says the build in CI is failing on this line. You'll need to `useAppDispatch` to get a reference to it.

Reply:
Of course!